httpx/_client.py

```diff
@@ -322,6 +322,10 @@ def redirect_url(self, request: Request, response: Response) -> URL:
 
         url = URL(location, allow_relative=True)
 
+        # Check that we can handle the scheme
+        if url.scheme and url.scheme not in ("http", "https"):
```

Comment:
This pattern `("http", "https")` (or something similar) is repeated in a few places, like:

https://github.com/jcugat/httpx/blob/9160631d7fb957a319ce8985c97a858f8eab49a3/httpx/_client.py#L609
https://github.com/jcugat/httpx/blob/9160631d7fb957a319ce8985c97a858f8eab49a3/httpx/_models.py#L105
https://github.com/jcugat/httpx/blob/9160631d7fb957a319ce8985c97a858f8eab49a3/httpx/_models.py#L132

It would probably make sense to unify it in a single place.

Reply:
In these kinds of cases, if it's a single-liner I'd often prefer *not* to introduce indirection. It's as clear as it can possibly be, and it's not a complicated expression.